File: LeetCode/1351.cpp
//Link: https://leetcode.com/problems/count-negative-numbers-in-a-sorted-matrix/

class Solution {
public:
    int countNegatives(vector<vector<int>>& grid) {
        int count = 0;
        for(auto row : grid)
            for(auto cell : row)
                if(cell < 0) count++;
        return count;
    }
};

File: LeetCode/1351.py
#Link: https://leetcode.com/problems/count-negative-numbers-in-a-sorted-matrix/

class Solution:
    def countNegatives(self, grid: List[List[int]]) -> int:
        count = 0
        for row in grid:
            for cell in row:
                if cell < 0 : count += 1
        return count
        